service/operator/service.go

```diff
@@ -123,6 +126,11 @@ func (s *Service) addFunc(obj interface{}) {
 		s.Logger.Log("error", "could not convert to azuretpr.CustomObject")
 	}
 
+	_, err := azure.NewClients(s.AzureConfig)
+	if err != nil {
+		s.Logger.Log("error", "could not create azure api clients '%#v'")
+	}
```

Comment:
The problem is the auth tokens expire. We can build renewing the token into the client but I think we can do that later.